src/_shared/js/images.js

```diff
@@ -18,18 +18,25 @@ export function addSourceset(responseJson) {
 
     return [highDef, lowDef];
    });
-   }
+}
 
-export function insertBetweenComments(sources) {
+export function insertSrcset(sources) {
 
-	let pictures = Array.from(document.getElementsByTagName('picture'));
+    let picture = document.getElementsByTagName('picture')[0];
 
-  return write(() => {
-		pictures.forEach((picture, index) => {
-      picture.insertBefore(sources[index][0], picture.firstChild);
-      picture.insertBefore(sources[index][1], picture.firstChild);
+    return write(() => {
+      sources.forEach((source, index) => {
+        picture.insertBefore(source[0], picture.firstChild);
```

Comment:
We're going to refactor a lot of this together with the multiple in a separate PR. Currently just getting our versions merged so we're not working in parallel.